packages/macros/src/index.ts

```diff
@@ -132,6 +127,11 @@ const plugin: UnpluginCombineInstance<Options | undefined> =
 
       options.plugins.vue,
       resolvePlugin(VueJsxDirective, framework, options.jsxDirective),
+      resolvePlugin(
```

Comment:
`VueReactivityTransform` cannot be executed after Vue plugin, since we need to rewrite `defineProps` in Vue SFC.

Reply:
Yes, I should support vue-sfc in jsx-directive.